### DISCUSSION TITLE
Which embedding model does Ollama web UI use to chat with PDF or Docs?

### DISCUSSION THREAD

**Discussion by karrtikiyer-tw** [open-webui/open-webui#551] [Q&A]
Can someone please share the details around the embedding model(s) being used? And if there is a provision to provide our own custom domain specific embedding model if need be?
Thanks,
Karrtik

**Comment by baumandm** [comment]
I looked into this recently, and AFAICT, it uses the ChromaDB default [[1]](https://docs.trychroma.com/embeddings/hugging-face), which is [sentence-transformers/all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2), a tiny 80MB model. 

There's a config option [here](https://github.com/ollama-webui/ollama-webui/blob/main/backend/config.py#L114) but it doesn't look like it's being used.
